2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,41 @@
 function populateTodoList(todos) {
```

Comment:
Please finish this exercise because your solution is not working. 
![image](https://user-images.githubusercontent.com/54252969/194707787-972f763e-c1e5-41a0-a04a-f9be8080c86d.png)